### DISCUSSION TITLE
Setting Splash Screen Background

### DISCUSSION THREAD

**Discussion by baskren** [unoplatform/uno#17839] [Q&A] [answered]
EDITED 

In pre-5.1 versions of Uno, the Splash Screen Background color could be specified via the `<UnoSplashScreen>` tag in the `base.props` file in the `[YOUR_PROJECT].Shared project`. Now, with the (much better) single project structure, the `base.props` file is no longer available.

Fortunately, the `<UnoSplashScreen>` tag is still available! In your single project's `[YOUR_PROJECT].csproj` file, you can add the following code:

It would be great if,  in the single project's `[YOUR_PROJECT].csproj` file, you could do something like add the following code:
```xml
      <ItemGroup>
        <UnoSplashScreen
          Include="$(MSBuildThisFileDirectory)Assets\Splash\splash_screen.svg"
          BaseSize="128,128"
          Color="#FFFFFF" />
      </ItemGroup>
```

A couple points of discussion:

1) The above works on WASM too!

2) DOCUMENTATION UPDATE REQUEST: Could the above be used replace or supplement the [Splash Screen Customization help page](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/features-splash-screen.html), since Google takes us there when searching on this topic?

3) Unfortunately, it appears that the `<UnoIcon>` tag no longer works (at least when doing an Android Release build).

**Comment by baskren** [comment]
OK, so it appears that it's not only `<UnoIcon>` that doesn't work in RELEASE builds, it's `<UnoSplashScreen>` as well.  Bummer.

**Comment by kazo0** [comment, marked as answer]
Hi @baskren!

With the move to single project, the `UnoImage` and `UnoSplashScreen` items can now be customized by some new MSBuild properties that are detailed here:

[UnoIcon customization](https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html?tabs=singleproject%2CAndroid#utilizing-sdk-properties)
 
[UnoSplashScreen customization](https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html?tabs=singleproject%2CAndroid#utilizing-sdk-properties-1)

Take a look at the Android specific documentation just below this as well:

![image](https://github.com/user-attachments/assets/30906281-d9f6-4ac5-bdac-01b6c44f45e5)

Hopefully that can help with your Android Release build

I will create some action items to take a look at the different splashscreen customization docs that we have and see what we can do to organize and make them discoverable

Let me know if you are blocked by anything else around this!

**Comment by baskren** [comment]
@kazo0 :  WOW!  Thank you for pointing these out!  And that also explains the issues I had with the Android Release builds.

Maybe your post will help elevate the SEO for these links.

**Comment by agneszitte** [comment]
Related issue to improve the current documentation: https://github.com/unoplatform/uno/issues/17850
